app/javascript/components/select/VSelectSearchable.vue

```diff
@@ -129,6 +129,9 @@ export default {
     },
     hasSelectedOption () {
       return this.selectedInternal !== UNDEFINED_OBJECT
+    },
+    defaultSearchTerm () {
```

Comment:
This is fine, but I think we can maybe handle this differently thinking about default vs none options and whether none is the same as nothing being selected etc. One to think about for updating the comp lib.